src/components/panels/SentientOutpostsPanel.vue

```diff
@@ -29,17 +34,23 @@ import TimeBadge from '@/components/TimeBadge.vue';
 import HubPanelWrap from '@/components/HubPanelWrap';
 import HubImg from '@/components/HubImg.vue';
 
-import moment from 'moment';
+//import moment from 'moment';
 
 import sentient from '@/assets/img/factions/sentient.svg';
 
 export default {
   name: 'SentientOutpostsPanel',
   props: ['sentientOutposts'],
+  watch: {
+    mission: function() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Insert `·`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5481243)

Reply:
fix it and i'll merge